### DISCUSSION TITLE
max_concurrent_with_start_early

### DISCUSSION THREAD

**Discussion by conandrum** [swar/Swar-Chia-Plot-Manager#1183] [Q&A]
Hi chia fans,

I am struggling with **max_concurrent_with_start_early** .

I have:
```
**global:**
max_concurrent: 24
max_for_phase_1: 12

**job:**
max_concurrent: 24
max_for_phase_1: 12
max_concurrent_with_start_early: 26
concurrency_start_early_phase: 4
```

**What I want to do:**
I am running 24 concurrent plots.
At some point in the process, I get 2-3 plots in phase#4 at the same time... 
At this same time my total in phase#1 is also lower than the max_for_phase_1.
I want 2 more plots to be allowed to be created on top of the max_concurrent (24) in order to reach 26.

However, I do not see this happening.
**What am I doing wrong here?**

Thanks people and wish you all the success in your CHIA projects.

UPDATE: posted this on reddit also: https://www.reddit.com/r/chia/comments/nwtn09/swar_max_concurrent_with_start_early/

**Comment by Out-Grow** [comment]
Change global max_concurrent: 24 to:

max_concurrent: 26

**global:**
max_concurrent: 26
max_for_phase_1: 12

**job:**
max_concurrent: 24
max_for_phase_1: 12
max_concurrent_with_start_early: 26
concurrency_start_early_phase: 4
